src/core/service.c

```diff
@@ -200,16 +200,28 @@ static void service_stop_watchdog(Service *s) {
         s->watchdog_timestamp = DUAL_TIMESTAMP_NULL;
 }
 
+static usec_t service_get_watchdog_usec(Service *s)
+{
```

Comment:
nitpick: place opening bracket `{` on the same line as the function name, also see CODING_STYLE.